airbyte-webapp/src/components/JobItem/components/AttemptDetails.tsx

```diff
@@ -69,37 +69,34 @@ const AttemptDetails: React.FC<IProps> = ({ attempt, className, configType }) =>
   const isFailed = attempt.status === Status.FAILED;
 
   return (
-    <div className={classNames(styles.details, className)}>
-      <div>
-        <span>{formatBytes(attempt?.bytesSynced)} | </span>
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.details}>
+        <span>{formatBytes(attempt?.totalStats?.bytesEmitted)}</span>
```

Comment:
Unsure if it's from another PR but formatBytes lacks an `|| 0` so you can get NaN bytes.

Reply:
Fixed in #15795